x/metadata/keeper/keeper_test.go

```diff
@@ -2,6 +2,7 @@ package keeper_test
 
 import (
 	"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
This should be moved down to 2nd to last (with the other testify entries)

Reply:
👍